flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/adaptive/allocator/VertexParallelism.java

```diff
@@ -30,6 +32,7 @@
  * SlotAllocator#tryReserveResources(VertexParallelism)}.
  */
 public interface VertexParallelism {
+    @Nonnull
```

Comment:
I think it's better to add some javadoc here.

Reply:
Sure.